.eslintrc

```diff
@@ -488,6 +488,26 @@
         ]
       }
     },
+    {
+      "files": [
+        "packages/activation-notice/**/*.js",
+        "packages/stories-block/**/*.js",
+        "packages/tinymce-button/**/*.js"
+      ],
+      "rules":{
+        "no-restricted-imports": [
+          "error",
+          {
+            "paths": [
+              {
+                "name": "react",
```

Comment:
Same for react-dom.

Might also want to do the same for encouraging using wordoress/i18n while at it

Reply:
I didn't do this, as gutenberg doesn't do this 
https://github.com/WordPress/gutenberg/blob/34fbec666d9b210e94df56246bc208da64d3ad10/.eslintrc.js#L69-L111

Reply:
This is not Gutenberg though. Let‘s add this please